arch/cpu/cc2538/dev/rom-util.h

```diff
@@ -57,7 +57,7 @@
 /** \name ROM utility function library API access table structure
  * @{
  */
-struct rom_util_api {
+typedef struct rom_util_api_t {
```

Comment:
The base struct should be named "rom_util_api" to distinguish it from the typedef name "rom_util_api_t".